example/ndpiReader.c

```diff
@@ -3901,6 +3901,9 @@ static void printResults(u_int64_t processing_time_usec, u_int64_t setup_time_us
       printf("\tPatricia protocols:   %llu/%llu (search/found)\n",
 	     (long long unsigned int)cumulative_stats.patricia_stats[NDPI_PTREE_PROTOCOLS].n_search,
 	     (long long unsigned int)cumulative_stats.patricia_stats[NDPI_PTREE_PROTOCOLS].n_found);
+      printf("\tPatricia protocols 6: %llu/%llu (search/found)\n",
```

Comment:
What about something like:

```suggestion
      printf("\tPatricia protocols IPv6: %llu/%llu (search/found)\n",
```

Reply:
Done